src/clients/discord/discord.voice.service.ts

```diff
@@ -108,6 +109,29 @@ export class DiscordVoiceService {
         this.disconnect();
       }
     });
+
+    const voiceChannelId = channel.id;
+    const intervalId = setInterval(async () => {
+      const voiceChannel = (await member.guild.channels.fetch(
+        voiceChannelId,
+      )) as VoiceChannel | undefined;
+      if (voiceChannel === undefined) {
+        clearInterval(intervalId);
+        return;
+      }
+      const voiceHumans = voiceChannel.members.filter(
+        (member) => !member.user.bot,
+      );
+      if (voiceHumans.size === 0) {
+        try {
+          this.stop(true);
+          this.disconnect();
+          clearInterval(intervalId);
+        } catch (e) {
+          this.logger.error(e);
```

Comment:
Log more than just the error. Where did something go wrong?

Reply:
how you mean?

Reply:
``this.logger.error('Unable to check for members for current channel: ' + error);``